### DISCUSSION TITLE
GitHub Runner App as a Service for Windows

### DISCUSSION THREAD

**Discussion by arjungoel** [community/community#65932] [Actions]
### Select Topic Area

Show & Tell

### Body

Hi GitHub, I was configuring Windows self-hosted runner and while configuring runner app as a service I noticed a weird thing. When I ran this command `Start-Service "actions.runner.*"` using Windows Powershell (Administrator) I expect the runner to be running as a service but when I checked the status using the following command `Get-Service "actions.runner.*"` I am seeing this as attached in the screenshot. 

I even troubleshooted it but no luck. Can you please confirm if that's a bug for Windows or not. I followed this link: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/configuring-the-self-hosted-runner-application-as-a-service?platform=windows

Please acknowledge.
![github-actions-runner-windows](https://github.com/community/community/assets/25618368/5bbd3b3c-4f08-4cc3-88b4-249e73a5b80b)

**Comment by aagashe-attekus** [comment]
Unbelievable that 2years passed this is NOT resolved.

**Comment by aagashe-attekus** [comment]
Ok GitHub here is the solution, would you care to update your 'documentation' !?

```
Would you like to run the runner as service? (Y/N) [press Enter for N] Y
User account to use for the service [press Enter for NT AUTHORITY\NETWORK SERVICE] azureuser
Password for the account windows-vs2022-\azureuser ****************
Granting file permissions to 'windows-vs2022-\azureuser'.
Service actions.runner.attekus-tech.windows-vs2022 successfully installed
Service actions.runner.attekus-tech.windows-vs2022 successfully set recovery option
Service actions.runner.attekus-tech.windows-vs2022 successfully set to delayed auto start
Service actions.runner.attekus-tech.windows-vs2022 successfully configured
Waiting for service to start...
Service actions.runner.attekus-tech.windows-vs2022 started successfully
```
azureuser is the local account.
